Lib/pathlib/_abc.py

```diff
@@ -234,7 +234,7 @@ def _parse_path(cls, path):
             elif len(drv_parts) == 6:
                 # e.g. //?/unc/server/share
                 root = sep
-        parsed = [sys.intern(str(x)) for x in rel.split(sep) if x and x != '.']
+        parsed = [x for x in rel.split(sep) if x and x != '.']
```

Comment:
I don't understand why you removed the `sys.intern` call here. Even if you wanted to speed up pickling, we should still strive to intern individual path components, as these are extremely likely to be shared between many path objects.